client/src/main/java/com/theokanning/openai/OpenAiService.java

```diff
@@ -72,25 +73,50 @@ public OpenAiService(final String token, final Duration timeout) {
      *
      * @param token   OpenAi token string "sk-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
      * @param timeout http read timeout, Duration.ZERO means no timeout
+     * @param orgId   specify which organization is used for an API request
+     */
+    public OpenAiService(final String token, final Duration timeout, final String orgId) {
```

Comment:
nit: maybe introduce a param with a builder that sets all configuration properties so that adding more fields in the future does not require change in 2 public methods.